server/controllers/carController.js

```diff
@@ -4,15 +4,15 @@ import CarModel from '../models/carModels';
 
 const _ = require('lodash');
 
-const carData = new CarModel();
+// const carData = new CarModel();
 
 function viewCarManager() {
-  return (req, res, next) => {
+  return async (req, res, next) => {
```

Comment:
You can abstract these controller methods to do one thing, that is, the control flow of data. They should receive data pass to it to another method (a service) which then returns data to this method that then responds.

E.g.
Instead of checking for status here, then fetching data from the db, then checking for maxprice and minPrice, this could be in another method.
So you'd pass the request data to another method that does the job and then return the response
```
async function viewCarManager(req, res, next) {
    const { status, min_price, max_price } = req.query;
    const result = await methodDoingAllTheWork(status, min_price, max_price);
    if (error in result) {
        return res.status(400).send({ your object here })
    }
    return res.status(200).send({ result })
}
```